### ISSUE TITLE
GHDB scraper produces inaccurate output

### ISSUE THREAD

**Issue by marz-hunter** [opsdisk/pagodo#96]
I found that the output from ghdb scraper was not precise. for example the title gives "Google Dork" but when viewed it produces site:"*.edu" intitle:"index of"|"*.db" and the tool saves the output "Google Dork" instead of site:"*.edu" intitle:" index of"|"*.db"

https://github.com/opsdisk/pagodo/assets/58464282/3518c55f-6156-4638-8861-38edd673bb18

**Comment by marz-hunter** [comment]
output tools. I think crawling a url like https://www.exploit-db.com/ghdb/8389 would be more accurate although it would take a little longer
![output](https://github.com/opsdisk/pagodo/assets/58464282/3ef86418-c15f-428a-8070-35c14ca303a2)

**Comment by marz-hunter** [comment]
crawling and then taking this part will be more accurate
![get](https://github.com/opsdisk/pagodo/assets/58464282/f8c292b9-6827-48e6-8491-8e1c5c9ec365)

**Comment by opsdisk** [comment]
Hi @marz-hunter - thanks for opening an issue.  In the past, I've also noticed the data isn't structured as precisely as it should be by exploit-db.com.  I vaguely recall thinking exploit-db.com needs to clean up the data instead of wanting to handle edge cases or invest any more time in the script.  Give me a week or two to dig deeper into it though.

In the mean time, you could reach out to them and see if they could clean up the ones you found.  Email can be found here: https://www.exploit-db.com/submit

![image](https://github.com/opsdisk/pagodo/assets/8660252/681b2b9c-4623-4e6f-9fa0-26d71b9d8c10)

**Comment by opsdisk** [comment]
Just found this as well that could be used https://gitlab.com/exploit-database/exploitdb/-/blob/main/ghdb.xml

Mind checking it for the same dependencies you found?

**Comment by marz-hunter** [comment]
yeah this one looks pretty good https://gitlab.com/exploit-database/exploitdb/-/blob/main/ghdb.xml, but updates seem to take a while

**Comment by opsdisk** [comment]
I'll keep this issue for a while to see if they update it.  Let me know if you do submit a PR against https://gitlab.com/exploit-database/exploitdb/-/blob/main/ghdb.xml
